### PR TITLE
Updating toast message when the user is attempting to create an account with an existing one

### PR DESCRIPTION
**Pull request by escobarjonatan**
Decided to create a new PR since the other one had too much going on

[Previous PR](https://github.com/projectNEWM/newm-artist-portal/pull/95)

### CODE REVIEW

**Comment by j-mateo** [comment]
Task linked: [CU-382cp84 Error toast message for social sign-up account email conflict](https://app.clickup.com/t/382cp84)

File: src/components/LinkedInLogin.tsx [hunk 1]
```diff
@@ -20,6 +21,14 @@ const LinkedInLogin: FunctionComponent = () => {
...
     redirectUri,
     scope: "r_liteprofile r_emailaddress",
+    onError: () => {
+      dispatch(
+        setToastMessage({
+          message: "Linkedin authentication was not successful.",
+          severity: "error",
+        })
+      );
+    },
```
[the remark this answers is not in the document]
Comment (escobarjonatan) [reply src/components/LinkedInLogin.tsx:31]: Good catch, I put a conditional to check for that error

File: src/modules/session/utils.ts [hunk 2]
```diff
@@ -31,3 +32,20 @@ export const handleLogout = (state: SessionState) => {
...
+      ? "The email for this account is already in use"
+      : "An error occurred while logging in";
+
+  dispatch(
+    setToastMessage({
+      message: errorMessage,
+      severity: "error",
+    })
+  );
+};
```
[the remark this answers is not in the document]
Comment (escobarjonatan) [reply src/modules/session/utils.ts:51]: This is actually one of the reasons why i didnt refactor the other error message. There are many ways to accomplish the same thing but i am not sure what the best approach for our scenario is. Went ahead and made it a thunk

**Comment by escobarjonatan** [comment]
Created [Toast Message | Make error states the same throughout our app](https://app.clickup.com/t/3r2kmyz)

**Pull request merged**